AuxTel/Standard-Operations/Daytime-Operations/latiss-daily-calibrations-BIAS-DARK-FLAT-all-filters-empty-procedure.rst

```diff
@@ -40,15 +40,14 @@ Post-Condition
 Procedure Steps
 ===============
 
-#. Calibrations images should only be taken if the LATISS WREB temperature is under temperature control. Check the LATISS WREB temperatures under the `AuxTel (LATISS) Temperatures and Pressures dashboard`_ in Chronograf. WREB temperatures are visible in the lower middle panel labeled *WREB On Board*. If the *mean_temp2* (top blue line) is between 26-29 degress C, the temperature is suitable for taking calibrations. During the daytime, the fan on the WREB board may not be sufficient to cool the WREB down to these temperatures, so during warmer months you may have to wait until later in the day or early in the morning for it to reach the desired temperature. If the temperature is too high, do not proceed to the next steps.
 #. Enable ATCS and LATISS using the standard scripts :file:`enable_atcs.py` and :file:`enable_latiss.py` with no configuration. 
 #. Enable ``Scheduler:2`` with a valid scheduler configuration. Use the standard script :file:`auxtel/scheduler/enable.py` with the configuration required for the run and available in the corresponding night log. 
 #. **Setup LATISS calibrations** BLOCK will setup ATCS and white light for calibrations. It enables and turns on the ATWhiteLight, enables OCPS:1 and commands AuxTel mount and dome to the FLAT position. Run the script :file:`add_block.py` to the ATQueue  with the following configuration:
 
     .. code-block:: text
       :caption: :file:`add_block.py`
 
-      id: setup_latiss_calibrations
+      id: BLOCK-309
```

Comment:
I wonder whether we should include a note /warning saying that ATWhitelight may be turned on even if the script fails or that the script fails to turn it on. And the scripts to be run in each scenario. What do you think? 

Perhaps it should go as a mini troubleshooting page instead of including the information here .....

Reply:
I hope that some of those failures will go away now that we have removed the monochromator from the block (this was only put in place on Friday Sept 13th). In the past, when we just ran the whitelight it was pretty reliable. Based on that I think a mini troubleshooting page might be better, maybe one for the calsys itself.